DSCResources/ProcessMitigation/ProcessMitigation.psm1

```diff
@@ -146,78 +187,418 @@ function Test-TargetResource
         [string]
         $MitigationTarget,
 
-        [Parameter()]
-        [string[]]
-        $Enable,
+        [Parameter(Mandatory = $true)]
+        [string]
+        $MitigationType,
```

Comment:
Because we have define an explicit list of MitigationNames and types in the schema.mof does it make sense to have that same list here?  I know the list is HUGE so maybe a using dynamic parameters could be used?